### PR TITLE
add invite icon and Survey Link TFAC 173/277

### PR DESCRIPTION
**Pull request by amaliwanag2**

### CODE REVIEW

**Comment by amaliwanag2** [comment]
the survey link ac is blocked by TFAC-294

File: src/pages/index.js [hunk 1]
```diff
@@ -398,7 +406,11 @@ const Index = ({ data: initialData }) => {
           ) : null}
           <div className={classes.fullContainer}>
             <div className={classes.topContainer}>
+              <Link className={classes.feedbackLink} to={surveyLink}>
```
Comment (kmjennison) [inline src/pages/index.js:409]: Please use a Button component for consistent styling & animation.

File: src/pages/index.js [hunk 2]
```diff
@@ -398,7 +406,11 @@ const Index = ({ data: initialData }) => {
...
+                <Typography>FEEDBACK</Typography>
+              </Link>
               <div className={classes.userMenuContainer}>
+                <InviteFriendsIconContainer user={user} />
```
Comment (kmjennison) [inline src/pages/index.js:413]: A few styling changes:
* Let's add a little more horizontal spacing around the icon so it matches the distance between other items in the menu
* This icon is bigger than the settings icon. Try 20x20px to match?
Screen Shot 2021-03-22 at 2 09 43 PM

File: src/components/InviteFriendsIcon.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+          <Typography className={classes.titleSection}>
+            You've recruited{' '}
+            <span style={{ fontWeight: 'bold' }}>
+              {`${numUsersRecruited} friend${
+                numUsersRecruited > 1 ? 's' : ''
+              } `}
+            </span>{' '}
+          </Typography>
+          <Typography variant="body2">
+            invite a friend! When they join, you'll earn a treat for 10 shelter
```
Comment (kmjennison) [inline src/components/InviteFriendsIcon.js:57]: Capitalize "invite"

**Review by kmjennison — CHANGES_REQUESTED** [review]

File: src/pages/index.js (hunk 2 above)
Comment (amaliwanag2) [reply src/pages/index.js:413]: 20x20 looked kind of off, the icon just seemed smaller than the settings but I can push and get your eyes on it

Comment (amaliwanag2) [reply src/pages/index.js:416]: I ended up going 24x24, which is the size of the paw icon.  The social share icon is weird tho as the svg doesn't use up all of the 24px vertical height, which is why going 20x20 looked weird.  LMK your thoughts.  Fixed the spacing

Comment (kmjennison) [reply src/pages/index.js:416]: Style feedback:
* I agree with you about the icon feeling small. Maybe split the difference and go 28px?
* The icon button is now elliptical but should be circular (see pic below). Make sure the margin is on the top-level element.
* Add some top margin on the popover so its top is in line with the other popovers in the menu
* Maybe `gutterBottom` on the popover text or some top/bottom margin around the invite friends component? Feels cramped.
Screen Shot 2021-03-22 at 8 02 21 PM

**Review by kmjennison — CHANGES_REQUESTED** [review]
See style comments.

File: src/pages/index.js (hunk 2 above)
Comment (amaliwanag2) [reply src/pages/index.js:416]: pushed some style updates. lmk what you think

Comment (kmjennison) [reply src/pages/index.js:416]: This looks great to me!

File: src/pages/index.js [hunk 4]
```diff
@@ -398,7 +407,13 @@ const Index = ({ data: initialData }) => {
           ) : null}
           <div className={classes.fullContainer}>
             <div className={classes.topContainer}>
+              <Button className={classes.feedbackLink}>
+                <Link to={surveyLink}>
```
Comment (kmjennison) [inline src/pages/index.js:411]: Move LInk outside of the button. Currently, you can click the button (around the text) and go nowhere.

File: src/pages/index.js [hunk 5]
```diff
@@ -398,7 +407,13 @@ const Index = ({ data: initialData }) => {
...
+                  <Typography>FEEDBACK</Typography>
```
Comment (kmjennison) [inline src/pages/index.js:412]: When this is the child of a Button, no need for Typography or all-caps.

**Review by kmjennison — CHANGES_REQUESTED** [review]
Just a small change with button functionality. Otherwise, looks great!

**Review by kmjennison — APPROVED** [review]
LGTM! 🍝

**Pull request merged**